bitnami/external-dns/templates/secret.yaml

```
@@ -37,6 +37,9 @@ data:
   {{- if and (eq .Values.provider "hetzner") (not .Values.hetzner.secretName) }}
   hetzner_token: {{ .Values.hetzner.token | b64enc | quote }}
   {{- end }}
+  {{- if eq .Values.provider "civo" }}
+  apiToken: {{ include "external-dns.civo-credentials" . | b64enc | quote }}
+  {{- end }}
```

I'm not sure we need the `external-dns.civo-credentials` helper here, we could directly set:

```suggestion
  {{- if eq .Values.provider "civo" }}
  apiToken: {{  .Values.civo.apiToken | b64enc | quote }}
  {{- end }}
```

---

No we don't. Fixed